samples/scala-counter/src/main/protobuf/counter/counterapi.proto

```diff
@@ -0,0 +1,135 @@
+syntax = "proto3";
+import "cloudstate/entity_key.proto";
+
+package com.example.counter.api;
+
+service CounterService {
+
+  rpc Initialize(InitializeRequest) returns (InitializeResponse) {}
```

Comment:
@ryanhanks Can't the api_request and api_response be inferred from parameter types and return values?

Reply:
To make sure I understand correctly, are you suggesting these rpc method definitions are redundant, given that these types are now specified by the handler implementation? I was envisioning the trait that the handler impl inherits from to be driven from the service definition, so specifying the types here is necessary in this approach in order to generate the underlying trait.

Wondering if I'm misunderstanding your question though..

Reply:
@ryanhanks No, rather the other way around—using the `rpc` definitions you can know what messages are api_requests and which ones are api_responses. I.e. the annotations might be redundant?

Reply:
Ah, gotcha. Yeah so we could annotate these method descriptors with a method option, and use that if we wanted to make it explicit, and then I think that could eliminate requiring the declarations on the request and response messages 👍

Reply:
:+1: